lib/matplotlib/testing/widgets.py

```diff
@@ -0,0 +1,66 @@
+"""
+========================
+Widget testing utilities
+========================
+
+Functions that are useful for testing widgets.
+See also matplotlib.tests.test_widgets
+"""
+
+import matplotlib.pyplot as plt
+
+try:
+    # mock in python 3.3+
+    from unittest import mock
+except ImportError:
+    import mock
+
+
+def get_ax():
```

Comment:
I am 50/50 on moving this function.

Reply:
Actually the name of this function does not reflect its body.

Reply:
Then you should give it a better name in testing. I assume that functions defined in tests are not part of the public api. So this change would be matplotlib-internal.